lib/contentful.js

```diff
@@ -103,22 +104,26 @@ function fetchByContentfulId(contentfulId) {
 
 /**
  * Returns Contentful entries with contentType in given contentTypes array.
- * TODO: Add pagination support, this only returns the first 100 (Contentful default limit).
  *
+ * @param {Object} query
  * @param {Array} contentTypes
  * @return {Promise}
  */
-function fetchByContentTypes(contentTypes) {
+function fetchByContentTypes(contentTypes, queryParams = {}) {
   logger.debug('fetchByContentTypes', { contentTypes });
+  const skipParam = queryParams.skip || 0;
+
   const query = module.exports.getQueryBuilder()
     .contentTypes(contentTypes)
+    .skip(skipParam)
     .orderByDescCreatedAt()
     .build();
+
   return module.exports.getClient().getEntries(query)
-    .then((res) => {
-      logger.debug('fetchByContentTypes count', res.total);
-      return res.items;
-    })
+    .then(res => ({
+      meta: utilHelper.getMeta(res.total, res.skip, res.limit),
```

Comment:
I think there is a good opportunity to make this response more declarative by calling a parsing function here:
```
// implementation
function parseContentfulResponse(res) {
    return {
        data: res.items,
        meta: utilHelper.getMeta(res.total, res.skip, res.limit),
    };
}

// usage
.then(res => module.exports.parseContentfulResponse(res))
```